sdk/src/client/secret/types.rs

```diff
@@ -179,7 +179,7 @@ impl TryFrom<InputSigningDataDto> for InputSigningData {
 impl From<&InputSigningData> for InputSigningDataDto {
```

Comment:
Same

Reply:
And others

Reply:
Same as above, would rather do in another PR